FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureBlobJobStore.cs

```diff
@@ -199,6 +250,11 @@ public async Task CompleteJobAsync(Job job, CancellationToken cancellationToken
                 // Remove job from active job list.
                 await _blobContainerClient.DeleteBlobAsync(GetJobBlobName(job, AzureBlobJobConstants.ActiveJobFolder), cancellationToken);
 
+                if (releaseLock)
+                {
+                    await TryReleaseJobLockAsync(cancellationToken);
```

Comment:
Will we use the returned value anywhere (true/false)?

Reply:
No. Currently the return value is not used as this lock is released when application exits. Might be useful in future.